src/messages/services/banner/index.js

```diff
@@ -57,12 +57,27 @@ function mutateMarkup(markup) {
     }
 }
 
-function fetcherB({ account, merchantId, amount, offerType, currency, style: { typeEZP } }) {
```

Comment:
So I can see that this function isn't new, but why is it call fetcherB? Can a comment be added here?

Reply:
`fetcherA` and `fetcherB` are labeled as such just fo the ramp. Eventually they will be condensed back into a single `fetcher`. They are not supposed to have a long lifespan in the code.